### ISSUE TITLE
Loss is Nan

### ISSUE THREAD

**Issue by lianxxx** [hguosc/visual_attention_consistency#5]
Hi, thank you for your code. I only used the weighted sigmoidcrossentropy loss to train a multilabel classification network and I used the poly schedule to adjust lr but got nan loss after 3 epochs. Why is the learning rate set to the initial LR every 3 epochs in your code?

**Comment by GuoleiSun** [comment]
Hi,

Can you try to use pytorch 0.3.1? It works if switching to 0.3.1. I also got nan when using pytorch 0.4.1
